tests/sentry/types/test_region.py

```diff
@@ -64,6 +57,7 @@ def test_validate_region(self):
             valid_region.validate()
 
     def test_json_config_injection(self):
+        _clear_global_regions()
```

Comment:
It's marginal because this is in `tests/`, but maybe this shouldn't be underscore-prefixed since we depend on it here.